modeling/modeling-mongodb-documents-with-native-go-data-structures.md

```diff
@@ -0,0 +1,164 @@
+# Modeling MongoDB Documents with Native Go Data Structures
+
+In the past few getting started tutorials, we explored the various ways to interact with MongoDB data using the Go programming language (Golang). In particular, we spent time exploring each of the [create](https://www.mongodb.com/blog/post/quick-start-golang--mongodb--how-to-create-documents), [retrieve](https://www.mongodb.com/blog/post/quick-start-golang--mongodb--how-to-read-documents), [update](https://www.mongodb.com/blog/post/quick-start-golang--mongodb--how-to-update-documents), and [delete](https://www.mongodb.com/blog/post/quick-start-golang--mongodb--how-to-delete-documents) (CRUD) operations, which are critical when building amazing applications.
```

Comment:
I would maybe add a link to the Go homepage (https://golang.org/) on the first use of "Go".